src/ui/settings/editing.rs

```diff
@@ -65,7 +65,7 @@ fn delete_preset(
 	field_presets.set(presets);
 }
 
-fn prefix_line(
+fn preset_line(
```

Comment:
No idea why I named this `prefix` instead of `preset`... don't drink and code kids!